contrib/system-setup.bat

```diff
@@ -0,0 +1,75 @@
+@echo off
+SETLOCAL ENABLEEXTENSIONS
+
+set output="c:\system-setup.log"

```

Comment:
Actually, I originally had it writing to the same directory as the batch file, but on "modern" Windows, the file wasn't showing up.  It was only by specifying the root that I was able to find the file.

The idea of writing to temp and then opening automatically is worth trying though.